packages/bun-lambda/runtime.ts

```diff
@@ -364,17 +364,15 @@ function formatHttpEventV2(event: HttpEventV2): Request {
       headers.append(name, value);
     }
   }
-  for (const [name, values] of Object.entries(event.queryStringParameters ?? {})) {
-    for (const value of values.split(",")) {
```

Comment:
The split on comma is very bad. I have parameters like: `?select=name,age` and this is certainly not meant to be a multi value on select.